site/daemon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Weave Flux Daemon
+menu_order: 90
+---
+
+# Weave Cloud Deploy
+
+Flux daemon (fluxd, aka flux agent) is at the heart of Weave Cloud Deploy, functionality allowing automation of application deployments and version control of cluster configuration.
+Version controlling of cluster manifests provides reproducibility and a historical trail of events.
+
+## Flux daemon responsibilities
```

Comment:
I like this structure -- it's plain and up-front what you should expect the daemon to do for you.

Reply:
:dancer: